core/lib/node_framework_derive/src/labels.rs

```diff
@@ -0,0 +1,98 @@
+use std::fmt;
+
+use syn::{spanned::Spanned as _, Attribute, Result};
+
+/// Context label, e.g. `ctx(crate = "crate")`.
+#[derive(Default)]
+pub(crate) struct CtxLabel {
+    /// Special attribute that marks the derive as internal.
+    /// Alters the path to the trait to be implemented.
+    pub(crate) krate: Option<syn::Path>, // `crate` is a reserved keyword and cannot be a raw identifier.
+    pub(crate) span: Option<proc_macro2::Span>,
+    pub(crate) task: bool,
+    pub(crate) default: bool,
+}
+
+impl fmt::Debug for CtxLabel {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        // For some weird reason, doc tests fail with the derived impl, stating that
+        // `syn::Path` does not implement `Debug`.
```

Comment:
That's because `Debug` is implemented for `syn::Path` only if [the `extra-traits` crate feature](https://docs.rs/syn/latest/syn/#optional-features) is enabled. Most probably, this feature is enabled indirectly in the workspace, but not when compiling doc tests for a specific crate.